docs/learn/learn-nominator.md

```diff
@@ -49,18 +49,91 @@ nomination. However, the election algorithm attempts to minimize this situation,
 occur often, so you should almost always see only a single active nomination per era. See the
 [section on Phragmén optimization](learn-phragmen.md#optimizations) for more details.
 
+### Bags-list
+
+Nominating accounts are placed in a semi-sorted list called bags-list. This sorting functionality is
+extremely important for the
+[long-term improvements](https://gist.github.com/kianenigma/aa835946455b9a3f167821b9d05ba376) of the
+staking/election system. Bags-list allows up to 50,000 nominators to set their _intention_ to
+nominate, of which, the stake of the top 22,500 nominators is considered for elections that
+eventually determine the active validators. The bags-list can be previewed on
+[Polkadot JS Apps > Network > Staking > Bags > All Bags](https://polkadot.js.org/apps/#/staking/bags).
+
+![Bags list](../assets/staking/bags-list.png)
+
+:::info Minimum DOT required to earn staking rewards
+
+Minimum DOT required to submit intent to nominate is 10 DOT, but the minimum active nomination
+required to earn staking rewards is dynamic and may be much higher, which can be viewed on
+[Polkadot JS Apps > Network > Staking > Targets page](https://polkadot.js.org/apps/#/staking/targets).
+
+:::
+
+![Minimum Active Nomination](../assets/staking/min-active-nomination.png)
+
+Bonding additional tokens or unbonding the staked tokens will automatically place the nominating
+account in the appropriate bag. While the system tries its best to ensure nominators are always
+represented in the correct bag, certain changes in bonded funds (e.g. a slash in the negative
+direction, or rewards in the positive direction) can cause an account to be in the wrong bag, and
+for scalability reasons the system will not automatically self-adjust.
+
+:::caution `bagsList.putInFrontOf` and `bagsList.rebag` extrinsics
+
+The nominator accounts in a bag are sorted based on their insertion order, not by their nomination
+stake. `bagsList.putInFrontOf` extrinsic can be issued to move up in the bag, which might be very
+useful for the accounts in the last bag eligible for receiving staking rewards. Also, balance
+changes due to staking rewards or slashing do not automatically re-bag the account. Whenever
+applicable, Polkadot JS Apps UI prompts the nominator account to rebag or move-up and the
+instructions are available in this
+[support article](https://support.polkadot.network/support/solutions/articles/65000181018-i-have-more-than-the-minimum-bonded-but-i-m-not-getting-rewards).
+
+:::
+
+To demonstrate how bags-list works, let's imagine a simple bag system with 7 accounts and 3 bags:
+
+Alice: 10 DOT, Bob: 11 DOT, Charlie: 15 DOT, Dave: 20 DOT, Eve: 100 DOT, Frank 1000 DOT, Georgina:
+2000 DOT
+
+Bag1: Max 2000, Min 1000 - Frank, Georgina
+
+Bag2: Max 1000, Min 20 - Eve, Dave
+
+Bag3: Max 20, Min 10 - Alice, Bob, Charlie
+
+The bags are iterated based on _insertion_ order, not _amount at stake_. So if only five nominating
```

Comment:
misleading: bags are ordered and are iterated from high to low, WITHIN a bag we use insertion order.